conda_forge_tick/settings.py

```diff
@@ -0,0 +1,106 @@
+    To configure a settings value, set the corresponding environment variable with the prefix `CF_TICK_`.
+    For example, to set the `graph_github_backend_repo` setting, set the environment variable
+    `CF_TICK_GRAPH_GITHUB_BACKEND_REPO`.
+
+    To access the current settings object, please use the `settings()` function.
+
+    Note: There still exists a significant amount of settings that are not yet exposed here.
+    All new settings should go here, and the other ones should eventually be migrated.
+    """
+
+    model_config = SettingsConfigDict(env_prefix=ENVIRONMENT_PREFIX)
+
+    conda_forge_org: str = Field("conda-forge", pattern=r"^[\w\.-]+$")
+    """
+    The GitHub organization containing all feedstocks. Default: "conda-forge".
+    If you change the field name, you must also update the `ENV_CONDA_FORGE_ORG` constant.
+    """
+
+    graph_github_backend_repo: str = Field(
+        "regro/cf-graph-countyfair", pattern=r"^[\w\.-]+/[\w\.-]+$"
+    )
+    """
+    The GitHub repository to deploy to. Default: "regro/cf-graph-countyfair".
+    """
+
+    graph_repo_default_branch: str = "master"
+    """
+    The default branch of the graph_github_backend_repo repository.
+    """
+
+    @property
+    def graph_github_backend_raw_base_url(self) -> str:
+        """
+        The base URL for the GitHub raw view of the graph_github_backend_repo repository.
+        Example: https://github.com/regro/cf-graph-countyfair/raw/master
+        """
+        return f"https://github.com/{self.graph_github_backend_repo}/raw/{self.graph_repo_default_branch}"
+
+    github_runner_debug: bool = Field(False, alias="RUNNER_DEBUG")
+    """
+    Whether we are executing within a GitHub Actions run with debug logging enabled. Default: False.
+    This is set automatically by GitHub Actions.
+    https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables
+    """
+
+    frac_update_upstream_versions: Fraction = 0.1
+    """
+    The fraction of feedstocks (randomly selected) to update in the update-upstream-versions job.
+    This is currently only respected when running concurrently (via process pool), not in sequential mode.
+    Therefore, you don't need to set this when debugging locally.
+    """
+
+    frac_make_graph: Fraction = 0.1
+    """
+    The fraction of feedstocks (randomly selected) to update in the make-graph job.
+    In tests or when debugging, you probably need to set this to 1.0 to update all feedstocks.
+    """
+
+
+_USE_SETTINGS_OVERRIDE: BotSettings | None = None
+"""
+If not None, the application should use this settings object instead of generating a new one.
+"""
+
+
+def settings() -> BotSettings:
+    """
+    Get the current settings object.
+    """
+    if _USE_SETTINGS_OVERRIDE:
+        return _USE_SETTINGS_OVERRIDE.model_copy()  # prevent side-effects
+    return BotSettings()
+
+
+def use_settings(s: BotSettings | None) -> None:
```

Comment:
Should we make this a context manager?

Reply:
Sounds like a good idea! I've done this here: 507b9284f71396298f57d4ac7f5b9f02cf7f60f0